connector_flow/README.rst

```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Connector-Flow
+===========
+
+This module provides a minimal framework for common
+data import/export tasks based on the OpenERP Connector. Tasks like parsing
+and writing a CSV file or uploading a file to an FTP server can be chained
+into task flows.
+
+At the moment every flow must a have a unique start. One task can trigger
+several tasks.
+
+The module adds a new menu item "Import/Export" under the Connector top-level
+menu where tasks and task flows can be configured. Tasks can be run from
+the "Run Task" wizard. If a task needs a file as input, a file can be uploaded
+in the wizard.
+
+The *connector_flow_example_{ftp,product}* modules provide pre-configured
+demo flows.
+
+This module was definitely inspired by the works of Akretion (file_repository)
+and Camptocamp (connector_file).
+
+A very brief tutorial can be found at http://blog.initos.com/?p=1220
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/{project_repo}/issues>`_.
```

Comment:
{project_repot} {module_name} and {version} are place holders and must be replaced with the appropriate values. 
* s/{project_repot}/connetor-interface 
* s/{version}/8.0
* s/{module_name}/connector_flow